rosidl_default_generators/rosidl_default_generators-extras.cmake.in

```diff
@@ -1,15 +1,14 @@
 # generated from
 # rosidl_default_generators/rosidl_default_generators-extras.cmake.in
 
+find_package("ament_cmake_core" REQUIRED)
```

Comment:
Does this need to be quoted? (because its an extras file ?)

Reply:
Doesn't need to be. The quotes are just "nice" since it highlights that it is a string literal and not a variable by any chance.

Reply:
It more like personal preference. I like the differentiation in my editor with different colors.

We could do that in general if other agree. The CMake introspection code should be fine with this:

https://github.com/ament/ament_tools/blob/669f335018341097f282097fba9627c560af43f0/ament_tools/package_types/cmake.py#L95

Reply:
Looking at the ROS2 code base, we quote only when evaluating variables. So I'm leaning toward removing the quotes here to be consistent and raise the question of "to quote or not to quote?" separately from this PR.

Reply:
I just removed the quotes. (And I don't care enough to bring this up for discussion.)